server/controllers/find/coursesController.test.ts

```diff
@@ -89,13 +86,24 @@ describe('CoursesController', () => {
 
     describe('when some but not all organisations are not found by the organisation service', () => {
```

Comment:
`when some but not all organisations are --not-- found by the organisation service`